client/my-sites/email/email-management/home/email-plan-subscription/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { CompactCard } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import { getRenewalPrice } from 'calypso/lib/purchases';
+import AutoRenewToggle from 'calypso/me/purchases/manage-purchase/auto-renew-toggle';
+import RenewButton from 'calypso/my-sites/domains/domain-management/edit/card/renew-button';
+import type { SiteDetails } from '@automattic/data-stores';
+import type { Purchase } from 'calypso/lib/purchases/types';
+
+import './style.scss';
+
+type EmailPlanSubscriptionProps = {
+	isLoadingPurchase: boolean;
+	purchase?: Purchase;
+	selectedSite: SiteDetails;
+};
+
+export const EmailPlanSubscription = ( {
+	purchase,
+	isLoadingPurchase,
+	selectedSite,
+}: EmailPlanSubscriptionProps ) => {
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	if ( ! purchase && isLoadingPurchase ) {
+		return (
+			<CompactCard className="email-plan-subscription__card email-plan-subscription__placeholder">
+				<div />
+				<div />
+				<div />
+			</CompactCard>
+		);
+	}
+
+	if ( ! purchase ) {
+		return null;
+	}
+
+	const todayTimestamp = new Date().setUTCHours( 0, 0, 0, 0 );
+	const expiryTimestamp = new Date( purchase.expiryDate ).getTime();
+	const hasSubscriptionExpired = todayTimestamp > expiryTimestamp;
+
+	const getDescription = () => {
+		const renewalPrice = getRenewalPrice( purchase );
+		const currencyCode = purchase.currencyCode;
+		const formattedRenewalPrice = formatCurrency( renewalPrice, currencyCode, {
+			stripZeros: true,
+		} );
```

Comment:
```suggestion
		const formattedRenewalPrice = formatCurrency( renewalPrice, purchase.currencyCode, {
			stripZeros: true,
		} );
```
Nit, but I guess we don't need the variable assignment for `currencyCode`